bom-2.332.x/pom.xml

```diff
@@ -25,6 +25,11 @@
                 <artifactId>bootstrap5-api</artifactId>
                 <version>5.2.0-1</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>credentials</artifactId>
+                <version>1143.vb_e8b_b_ceee347</version>
+            </dependency>
```

Comment:
What is this for? It should be inheriting the version from the 2.346.x line.

Reply:
Thanks for the education.  I had forgotten that the 2.346.x line is taken into the 2.332.x line.  I'll remove that wasted addition in a later pull request.

Reply:
Thanks!